external_push/admin.py

```diff
@@ -13,4 +13,8 @@ class BrewersFriendPushTargetAdmin(admin.ModelAdmin):
 
 @admin.register(BrewfatherPushTarget)
 class BrewfatherPushTargetAdmin(admin.ModelAdmin):
-    list_display = ('gravity_sensor_to_push', 'status', 'push_frequency')
\ No newline at end of file
+    list_display = ('gravity_sensor_to_push', 'status', 'push_frequency')
+    
+#@admin.register(ThingSpeakPushTarget)
```

Comment:
Out of curiosity, why did you  comment out the admin?

Reply:
I forgot to comment when debugging... Sorry - needed to update this line also
`    list_display = ('name', 'push_frequency', 'api_key', 'brewpi_to_push')`